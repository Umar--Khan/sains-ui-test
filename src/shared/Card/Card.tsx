import { PropsWithChildren } from 'react';
import styled from 'styled-components';

import * as colors from 'src/styles/_colors.module.scss';

const StyledCard = styled.div`
  padding: 1.5rem;
  background-color: ${colors.white};
  margin: 1rem 0;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
`;

const Card = ({ children, ...rest }: PropsWithChildren<any>) => (
  <StyledCard {...rest}>{children}</StyledCard>
);

export default Card;
