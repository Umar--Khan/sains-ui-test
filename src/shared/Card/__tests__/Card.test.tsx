import renderer from 'react-test-renderer';

import Card from '../Card';

describe('<Card />', () => {
  it('Should render correctly', () => {
    const component = renderer.create(<Card>Hello</Card>);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
