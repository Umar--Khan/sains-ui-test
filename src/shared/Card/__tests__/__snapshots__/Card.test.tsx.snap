// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> Should render correctly 1`] = `
<div
  className="sc-bdfBwQ fvoACg"
>
  Hello
</div>
`;
